src/main/resources/swagger/api-docs.yaml

```diff
@@ -5621,6 +5625,59 @@ definitions:
         type: string
         description: The fully qualified name of the GCP service perimeter to put this project into in the form accessPolicies/[POLICY NUMBER]/servicePerimeters/[NAME]. Caller must have the add_project action for this service perimeter in Sam.
 
+  CreateWorkspaceResponse:
+    description: payload returned from create- or clone-workspace
+    required:
+      - attributes
+      - authorizationDomain
+      - bucketName
+      - createdBy
+      - createdDate
+      - isLocked
+      - lastModified
+      - name
+      - namespace
+      - workspaceId
```

Comment:
When I create a workspace in orchestration I get the following:
```
{
  "attributes": {},
  "authorizationDomain": [],
  "bucketName": "fc-<REDACTED>",
  "createdBy": "ansinghfirecloud@gmail.com",
  "createdDate": "2019-10-23T19:18:58.344Z",
  "isLocked": false,
  "lastModified": "2019-10-23T19:18:58.344Z",
  "name": "test-ws-anu",
  "namespace": "abbbbbanu",
  "workflowCollectionName": "<REDACTED>",
  "workspaceId": "<REDACTED>"
}
```
which includes a `workflowCollectionName` that isn't defined in this reponse

Reply:
it's defined as a property, but not `required` since it's an `Option` in the underlying case class - and it shows up in swagger ui

Reply:
oh! I see :)